l5kit/setup.py

```diff
@@ -7,8 +7,8 @@
 setup(
     name="l5kit",
     version=__version__,
-    description="Lyft Autonomous Vehicle Research library",
-    author="Lyft Level 5",
+    description="Level-5 Autonomous Vehicle Research library",
+    author="Level 5",
     author_email="l5kit@lyft.com",
```

Comment:
wondering if this has changed too

Reply:
Good question, for the moment I think they are still redirecting, I will double-check that when we change the repo org.